react/w-books/src/app/components/Form/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode, Component } from 'react';
+
+interface FormProps {
+  handleSubmit?: (event: React.FormEvent) => void;
+  children: ReactNode;
+  className?: string;
+}
+
+export default class Form extends Component<FormProps> {
```

Comment:
I think that is dont necessary use the class component, use the functional component and if you need some lifecycle method, use React Hooks.